dot/sync/disjoint_block_set_integeration_test.go

```diff
@@ -29,12 +25,12 @@ func TestDisjointBlockSet(t *testing.T) {
 	require.Equal(t, 1, s.size())
 
 	expected := &pendingBlock{
-		hash:    hash,
-		number:  number,
-		clearAt: time.Time{}.Add(ttl),
+		hash:   hash,
+		number: number,
 	}
 	blocks := s.getBlocks()
 	require.Equal(t, 1, len(blocks))
+	blocks[0].clearAt = time.Time{}
```

Comment:
we should assert the `clearAt` field somehow (even if it's not an exact assertion, at least verify it's set in the future or something)